src/main/java/com/autotune/analyzer/services/CreateExperiment.java

```diff
@@ -121,7 +121,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                         sendErrorResponse(response, null, HttpServletResponse.SC_BAD_REQUEST, addedToDB.getMessage());
                     }
                 } else {
-                    LOGGER.error("Failed to create experiment: {}", invalidKruizeObject.getValidation_data().getMessage());
+                    LOGGER.debug("Failed to create experiment: {}", invalidKruizeObject.getValidation_data().getMessage());
```

Comment:
this is actual error and need logging 
coz we sending error response to user .. so let it be error itself